src/registrar/admin.py

```diff
@@ -641,6 +660,18 @@ def formfield_for_manytomany(self, db_field, request, **kwargs):
             kwargs["queryset"] = models.Website.objects.all().order_by("website")  # Sort websites
         return super().formfield_for_manytomany(db_field, request, **kwargs)
 
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
+        # Removes invalid investigator options from the investigator dropdown
+        if db_field.name == "investigator":
+            kwargs["queryset"] = User.objects.filter(is_staff=True)
+            return db_field.formfield(**kwargs)
+        return super().formfield_for_foreignkey(db_field, request, **kwargs)
```

Comment:
Given that we have to do this, it implies that investigators are "invalid" if they don't have the role is_staff. Should we also make a model level change as well to throw an error on create/save, if investigator is not is_staff?

I took the conservative approach for this PR and did not include that change (as it wasn't specified in the ticket).

Reply:
I think this is fine. Right now, if you remove is_staff from a user the application that she's investigator on will not throw errors. They'll show blank for investigator because you are excluding non-staff from the dropdown but in the DB the investigator is still assigned. 

I think that's OK, just need to make sure the team knows about this.